.github/CHANGELOG.md

```diff
@@ -1124,182 +1127,4 @@
-5. Removed remaining references to Manston (EGMH) in agreements and in adjacent airport setup - thanks to @hsugden (Harry Sugden)
-6. Added EG R219 - thanks to @samw (Sam White)
-7. Added EGGP and EGPH GND/TWR/APP sector rings to display to Area - thanks to @lpeters (Luke Peters)
-8. Added EGCC GND/TWR/APP sector rings to display to Area - thanks to @hsugden (Harry Sugden)
-9. Added adjacent airports ownership for Scottish positions (000A Show adjacent departure airports) - thanks to @hsugden (Harry Sugden)
-10. Corrected Stornoway (EGPO) top-down priority - thanks to @hsugden (Harry Sugden)
-11. Added FL245 shelf/portion of the EGPX/EISN FIR - thanks to @hsugden (Harry Sugden)
-12. Fixed Lossiemouth/Kinloss MATZ airspace to fit ARTCC low naming conventions - thanks to @hsugden (Harry Sugden)
-13. Added EGPF GND/TWR/APP sector rings to display to Area - thanks to @lpeters (Luke Peters)
-14. Added Southern CTA bases and moved Clacton CTA base - thanks to @lpeters (Luke Peters)
-15. Added missing fixes on Gatwick STARs (TIMBA1J/1K/3F/2G) - thanks to @lpeters (Luke Peters)
-16. Added EGPD GND/TWR/APP sector rings to display to Area - thanks to @lpeters (Luke Peters)
-17. Added LTC SE and SW sector static boundaries - thanks to @samw (Sam White)
-18. Added AC Daventry sector static boundary - thanks to @hsugden (Harry Sugden)
-19. Added LTC NE and NW sector static boundaries - thanks to @hsugden (Harry Sugden)
-20. Reformatted RMA whitespace - thanks to @samw (Sam White) and @hsugden (Harry Sugden)
-21. Added missing Y3 airspace base (to the east of the EGGD CTA) - thanks to @hsugden (Harry Sugden)
-22. Added EGPD COPX agreements - thanks to @lpeters (Luke Peters)
-23. Added Essex Radar to Stansted Director automatic handoff - thanks to @samw (Sam White)
-24. Added missing agreements to/from PEAPP - thanks to @hsugden (Harry Sugden)
-25. Removed Manchester stand 50 - thanks to @acrix (Matthew Moy)
-26. AIRAC - Removed Manchester holding point W2 - thanks to @acrix (Matthew Moy)
-27. Adjusted CC INT/FIN sectors and inbound agreements with MAN - thanks to @hsugden (Harry Sugden)
-28. Added AC Dover sector static boundary - thanks to @hsugden (Harry Sugden)
-29. Added AC Clacton sector static boundary - thanks to @hsugden (Harry Sugden)
-30. Added AC Worthing sector static boundary - thanks to @hsugden (Harry Sugden)
-31. Updated French CTRs, added the Lille CTR, and updated the Deauville and Lille TMAs - thanks to @pierr3 (Pierre Ferran)
-32. AIRAC - CAE NDB withdrawn; fix FERAS withdrawn; EG D323A danger area updated - thanks to @hsugden (Harry Sugden)
-33. AIRAC - EGHC RWY 07 RNAV approach and associated airfield fixes - thanks to @acrix (Matthew Moy)
-34. Adjusted and renamed EGCC RMAs, including a single 4 mile ring centred on the ARP - thanks to @acrix (Matthew Moy)
-35. Added missing agreements between DUBN and Antrim, particularly for airway Y911 - thanks to @hsugden (Harry Sugden)
```

Comment:
seem to have cut off a bunch of lines in the changelog - could you check out adding them back?

Reply:
Strange, I forked it again before working on this and it seems cut off mid-way through a line...   Have copy and pasted the master back in to complete it in the branch.